src/Qir/build_all.ps1

```diff
@@ -0,0 +1,3 @@
+& Runtime\build-qir-runtime.ps1
+& Tests\build-qir-tests.ps1
+& Samples\build-qir-samples.ps1
```

Comment:
This and `test_all.ps1` are also just for myself, rather raw files (they don't analyze the result of the command), they list the commands to run manually.
I suspect the `qir-utils.ps1` is for the same or similar purpose, feel free to refer me to how to use it (some quick start/examples).